CHANGELOG.md

```diff
@@ -9,6 +9,37 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.6.0 - 2020-04-29
+
+### Added
+
+- Expose Prometheus metrics for Agent and Controller using the "/metrics" apiserver endpoint; "enablePrometheusMetrics" must be set to true in configuration.
+- Add documentation for deploying Prometheus and scrape Antrea metrics, along with sample YAML manifest.
+- Install portmap CNI by default in order to support Pods with "hostPort" set.
+- Support configurable ports for Agent and Controller apiservers.
+- Set default CPU resource requests for Antrea components in YAML manifest.
+- Add "/ovsflows" API endpoint to Agent to query OVS flows and "antctl get ovsflows" command; flows can be filtered by Pod / NetworkPolicy / OVS Table.
+- Add object type aliases to antctl (plural form and short alias).
+- Add namespace and name parameters to "antctl get networkpolicy".
+- Document known issues when deploying Antrea on Photon OS or CoreOS.
+
+### Changed
+
+- Add authentication to agent apiserver to enable external access (from outside of Agent Pod), and generate bearer token for local access instead of delegating authentication to K8s apiserver.
```

Comment:
API external access and antctl remote execution can be a feature?

Reply:
I think it's important to mention it, as it can impact how other software can integrate with Antrea and consume the Agent APIs

Reply:
Agreed. I meant do you want to add remote access/execution as an "Added" feature.

Reply:
I think it's a change to how we can access the APIs, but I guess there can be a blurry line between the 2. You would prefer for me to move it?

Reply:
I feel remote access and esp. remote Agent antctl sound like a feature.
But no strong opinion. Up to you.

Reply:
Happy to indulge, will make the change before I merge

Reply:
My mistake - we do not really support remote Agent command yet right? Then I would agree remote API call might not be a new feature. Again up to you to decide.

Reply:
Happy to leave it as a new feature

Reply:
Actually I'm going to follow my first instinct and leave it in "Changed" since it doesn't directly impact end users right now.